include/zephyr/bluetooth/uuid.h

```diff
@@ -2246,6 +5035,8 @@ struct bt_uuid_128 {
 #define BT_UUID_FTP                   BT_UUID_DECLARE_16(BT_UUID_FTP_VAL)
 #define BT_UUID_HTTP_VAL              0x000c
 #define BT_UUID_HTTP                  BT_UUID_DECLARE_16(BT_UUID_HTTP_VAL)
+#define BT_UUID_WSP_VAL               0x000e
+#define BT_UUID_WSP                   BT_UUID_DECLARE_16(BT_UUID_WSP_VAL)
```

Comment:
Just curious since you are adding `0x000e` here: What about `0x000b` and `0x000d`? Or don't they exist?

Reply:
Protocols identifiers '0x000b', '0x000d', '0x0013', '0x0015', '0x0018', '0x001a', '0x001c', '0x001d' don't exist.